doc/sphinx/dials_scale_user_guide.rst

```diff
@@ -19,60 +21,98 @@ The user is also advised to familiarise themselves with the standard program
 output, which may contain useful information, and the html report generated
 by scaling, which provides numerous plots relating to the merging statistics.
 
-Byte-sized guide to common scaling options
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To get started with using DIALS to scale data, load these 8 bits of information
-into memory, which cover the most commonly used options (with example values)
+Guide to common scaling options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+These sections cover the most commonly used options (with example values)
 for scaling routine macromolecular crystallography datasets.
 
-**When less is more - cutting back data.** It is often the case that not all
-of the data are suitable for merging, so
-try these options to get the best of your data:
-
-- :samp:`d_min=2.0`  Apply a resolution cutoff with this simple command.
-- :samp:`exclude_images="0:100:120"`  Remove a section of images, perhaps due to
-  bad per-image R-merge values as result of radiation damage. Use multiple
-  commands like this to exclude multiple ranges. The syntax is "a:b:c",
-  where a is the experiment number (starting at :samp:`0`), b is the initial image
-  to exclude and c is the final image to exclude.
-- :samp:`exclude_datasets="10 50 79"`  Go all out and remove whole datasets, based on
+**Cutting back data**
+After inspecting the statistics in the :samp:`dials.scale.html` file, such as
+the R-merge vs batch plot, it is often the case that not all of the data are
+suitable for merging, perhaps due to radiation damage or nonisomorphism.
+This can be the case within a  single sweep or across multiple sweeps in a
+multi-sweep/multi-crystal experiment. These are example options to use:
+
+- :samp:`d_min=2.0`  Applies a resolution cutoff at the given resolution (in Angstrom).
+- :samp:`exclude_images="100:120"`  Removes a section of images for a single
+  sweep dataset. Multiple commands like this can be used to exclude multiple ranges.
+  In the case of multiple-sweeps, one must also provide the experiment id that
+  the exclusion should apply to, with the syntax :samp:`exclude_images="a:b:c"`
+  where `a` is the experiment id (a number starting at :samp:`0`), `b` is the initial
+  image to exclude and `c` is the final image to exclude.
+- :samp:`exclude_datasets="10 50 79"`  Removes whole datasets, based on
   the dataset number; useful for large multi-crystal datasets.
```

Comment:
```suggestion
  the dataset number; useful for large multi-crystal datasets.
```
is "dataset number" distinct from "experiment ID"? If not, I would suggest ensuring consistency in terminology.